app/src/main/java/org/mozilla/fenix/utils/ClipboardHandler.kt

```diff
@@ -38,8 +38,7 @@ class ClipboardHandler(val context: Context) {
     val url: String?
         get() {
             return text?.let {
-                val finder = WebURLFinder(it)
-                finder.bestWebURL()
+                if(URLStringUtils.isURLLike(it)) it else null
```

Comment:
```suggestion
                if (URLStringUtils.isURLLike(it)) { it } else { null }
```
should fix lint issue